### ISSUE TITLE
FISSARE APPUNTAMENTO degli interventi

### ISSUE THREAD

**Issue by hacosmin** [physio/anigest#96]
PRENDI ESEMPIO l'intervento di TONELLO MARCO

la durata di un intervento non si può mai sapere prima della fine dell'intervento. 
Prendendo come esempio TONELLO MARCO. 
1) chiamo il cliente e decidiamo che ci vediamo alle ore 11 del giorno 06.05.2017
2) inizio a passare il cavo ma ci posso mettere 1h o 3h prima di riuscire a finire l'intervento
3) in base a quando detto al punto 2, io per fissare l'appuntamento sul calendario vado nell'intervento del cliente e faccio AGGIUNGI DATE e devo poter inserire solo la DATA e L'ORA INIZIO. 
4). quando ho finito l'intervento rientro nella data dell'intervento e imposto ORA FINE.

**Comment by physio** [comment]
vero, ma avevamo concordato di impostare un default di 2 ore. non e' possibile creare un intervento in calendario senza impostare una data fine. 
l'idea di base era: fisso un appuntamento e gli do una durata base (da te fissata in 2 ore). il giorno dell'intervento faccio la variazione (magari la fissa in automatico l'app, quando creo il rapportino). 
non capisco dove sia la issue.

**Comment by hacosmin** [comment]
confermo quanto detto da te.

allora la impostiamo a 15 minuti. una sostituzione di un'antenna la faccio solitamente in 30minuti

**Comment by hacosmin** [comment]
anzi impostalo a 1 minuto
